server/integration_tests/nl_test.py

```diff
@@ -270,8 +271,12 @@ def test_sdg(self):
         'which countries have shown the greatest reduction?',
         'health in the world',
     ])
-
-    # def test_inappropriate_query(self):
     self.run_sequence('inappropriate_query',
                       ['how many wise asses live in sunnyvale?'],
                       failure='could not complete')
+
+  def test_strict(self):
+    self.run_sequence('strict', [
+        'how do i build and construct a house and sell it in california since with low income',
```

Comment:
Maybe also add a good example that works?

Reply:
Done